File: Orientacao_Objetos/Conceitos/Orientacao_Objetos/src/exemplo01/Pessoa.java
package exemplo01;

import javax.swing.JOptionPane;

public class Pessoa {
	
	String nome;
	int idade;
	double altura;
	
	void obterDados() {
		 nome = JOptionPane.showInputDialog("Informe seu nome");
		 idade = Integer.parseInt(JOptionPane.showInputDialog("Informe sua idade"));
		 altura = Double.parseDouble(JOptionPane.showInputDialog("Informe sua altura"));
	}
	
	void retornarDados() {
		String mensagem = "Nome: " + nome;
			   mensagem+= "\nIdade: " + idade;
			   mensagem+= "\nAltura: " + altura;
			   
		JOptionPane.showMessageDialog(null, mensagem);
	}
	
}

File: JSP/Conceitos/src/main/webapp/pessoa.js
function valida() {
	let nome = document.getElementById("nome").value;
	let idade = parseInt(document.getElementById("idade").value);
	
	if (nome == "") {
		alert("Informe o nome");
		return false;
	} else if (idade < 0 || idade > 120) {
		alert("Idade inválida");
		return false;
	}
}